example.json

```diff
@@ -1,3 +1,15 @@
+// course.json
+"_extensions": {
+  "_visua11y": {
+    "_navOrder": 100,
+    "navLabel": "Accessibility",
+    "_navTooltip": {
+      "_isEnabled": true,
+      "text": "Visual accessibility settings"
```

Comment:
Although `_navTooltip text` is already set to the same value as `navigationAriaLabel`, for [core navigation](https://github.com/adaptlearning/adapt-contrib-core/blob/1e73f329c21e6eb906c102db23f600666117cd74/schema/course.model.schema#L369) the _ariaLabel_ is inherited as the default.
When editing button text, `aria-label`s are often forgotten about so I think it would be good to do the same as core here and keep the label and aria in sync by default. 

Note, I know we don't currently have the new schemas (mentioned [here](https://github.com/cgkineo/adapt-visua11y/pull/91#issuecomment-1602221262)) but I'll raise this as a separate issue.